packages/amplify-codegen/src/walkthrough/questions/selectProfile.js

```diff
@@ -0,0 +1,15 @@
+const inquirer = require('inquirer');
```

Comment:
Can you remove this. I don't see this being used anywhere

Reply:
Removing it